src/octree/node.rs

```diff
@@ -291,15 +291,17 @@ pub enum PositionEncoding {
     Uint8,
     Uint16,
     Float32,
+    Float64,
 }
 
 impl PositionEncoding {
     pub fn new(bounding_cube: &Cube, resolution: f64) -> PositionEncoding {
-        let min_bits = (f64::from(bounding_cube.edge_length()) / resolution).log2() as u32 + 1;
+        let min_bits = (bounding_cube.edge_length() / resolution).log2() as u32 + 1;
         match min_bits {
             0...8 => PositionEncoding::Uint8,
             9...16 => PositionEncoding::Uint16,
-            _ => PositionEncoding::Float32,
+            17...24 => PositionEncoding::Float32, // 24 because of the float mantissa
```

Comment:
Please add the diameter of the node for the 1. mm resolution here as a comment. This would have given me more context.

Reply:
Done.